common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/packets/WorldPackets.java

```diff
@@ -291,7 +291,7 @@ public void handle(PacketWrapper wrapper) throws Exception {
                         wrapper.clearInputBuffer();
                         // First set this packet ID to Block placement
                         wrapper.setId(0x08);
-                        wrapper.write(Type.LONG, -1L);
+                        wrapper.write(Type.POSITION, new Position((long) -1, (long) -1, (long) -1));
                         wrapper.write(Type.BYTE, (byte) 255);
```

Comment:
this could also be mapped to an unsigned int (as it is read so in the 1.8 server as well)

Reply:
https://github.com/ViaVersion/ViaVersion/blob/b04acf98c7b4a6c027a903ee6c088a123e828a8f/common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/packets/WorldPackets.java#L331-L343 it would need changing here too

Reply:
yep

Reply:
wait it already _is_ in the abstraction branch